cosmwasm/utils.js

```diff
@@ -66,8 +66,9 @@ const instantiateContract = (client, wallet, initMsg, config, { contractName, sa
                       initMsg,
                       contractName,
                       initFee,
+                      { admin: admin || account.address },
```

Comment:
we should default the admin to the governance address if it's not overridden, not the deployer wallet

Reply:
done